migrations/helpers/DatabaseTaskTemplate.js

```diff
@@ -0,0 +1,67 @@
+class DatabaseTaskTempalte {
+	constructor({
+		id,
```

Comment:
duplication of parameters seems not to be neccessary, because either `id` or `_id` will be used to set `this.id`. same for other constructor parameters

Reply:
If you fetch and pass data from the DB you have _id and it is a nice to have, that you can pass it directly. Same by already defined database tasks. But for human is is easyer to write and ready without _ and $.